test/Interop/Cxx/static/Inputs/static-var.h

```diff
@@ -0,0 +1,43 @@
+static int staticVar = 2;
+inline void setStaticVarFromCxx(int newVal) { staticVar = newVal; }
+inline int getStaticVarFromCxx() { return staticVar; }
+
+inline int inlineMakeStaticVar() { return 8; }
+static int staticVarInlineInit = inlineMakeStaticVar();
+
+int makeStaticVar();
+static int staticVarInit = makeStaticVar();
+
+static const int staticConst = 4;
+
+inline int inlineMakeStaticConst() { return 16; }
+static const int staticConstInlineInit = inlineMakeStaticConst();
+
+int makeStaticConst();
+static const int staticConstInit = makeStaticConst();
+
+constexpr int makeStaticConstexpr() { return 32; }
+static constexpr int staticConstexpr = makeStaticConstexpr();
+
+class NonTrivial {
+public:
+  // TODO: Call `NonTrivial(int)` constructor once apple/swift/pull/30630 is
+  // merged.
```

Comment:
That PR is only enabling Swift code to call such constructors.

In fact, code below already calls `NonTrivial(1024)` -- so I don't understand what this TODO suggests to do.

Reply:
Clarified the todo.